apps/web/views.py

```diff
@@ -35,21 +35,30 @@ def internal_server_error(request):
 
 
 @throttle_classes([AnonRateThrottle, ])
-@api_view(['POST', ])
+@api_view(['POST', 'GET'])
 @permission_classes((permissions.AllowAny,))
 def contact_us(request):
     try:
-        user = User.objects.get(username=request.user)
-        name = user.username
-        email = user.email
-        request_data = {"name": name, "email": email}
-        request_data['message'] = request.data['message']
-        serializer = ContactSerializer(data=request_data)
-        if serializer.is_valid():
-            serializer.save()
-            response_data = {'message': 'Your message has been successfully recorded. We will contact you shortly.'}
-            return Response(response_data, status=status.HTTP_201_CREATED)
-        return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+        if request.method == 'POST':
+            user = User.objects.get(username=request.user)
+            name = user.username
+            email = user.email
+            request_data = {"name": name, "email": email}
+            request_data['message'] = request.data['message']
+            serializer = ContactSerializer(data=request_data)
+            if serializer.is_valid():
+                serializer.save()
+                response_data = {'message': 'Your message has been successfully recorded. We will contact you shortly.'}
+                return Response(response_data, status=status.HTTP_201_CREATED)
+            return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+
+        elif request.method == 'GET':
+            user = User.objects.get(username=request.user)
```

Comment:
@TroJan Thank you for guiding. But I think in this case we don't need to catch it separately as it is already being cached in the `try` and [except](https://github.com/Cloud-CV/EvalAI/blob/master/apps/web/views.py#L53) case.
The function works as follows:
1. If the user is logged in:
 a) On `get` request the `username` and `email` is being fetched from the database.
b) On `post` request the `name`, `email` and `message` is posted to the database.

2. If the user is not logged in then `name`, `email` and `message` is posted to the database.

Please review and tell me if I am missing somewhere ?